doc/manual/configuration.rst

```diff
@@ -774,6 +774,8 @@ packages that are built from a library: a ``-target`` packet that has the
 shared libraries needed during runtime and a ``-dev`` packet that has the
 header files and other needed files to link with this library.
 
+Bob does not support multiple or nested multiPackages in a single recipe, yet.
```

Comment:
What are multiple multiPackages?! Can you give me a example of what exactly you want to do/achieve?